drake/examples/Quadrotor/CMakeLists.txt

```diff
@@ -1,9 +1,17 @@
+include_directories(${PROJECT_SOURCE_DIR}/core)
+include_directories(${PROJECT_SOURCE_DIR}/solvers)
+include_directories(${PROJECT_SOURCE_DIR}/systems)
+include_directories(${PROJECT_SOURCE_DIR}/systems/frames)
+include_directories(${PROJECT_SOURCE_DIR}/systems/plants)
+include_directories(${PROJECT_SOURCE_DIR}/systems/plants/joints)
+include_directories(${PROJECT_SOURCE_DIR}/systems/controllers)
+include_directories(${PROJECT_SOURCE_DIR}/util)
+
 if (LCM_FOUND AND NOT WIN32)
   add_executable(runQuadrotorDynamics runDynamics.cpp)
-  target_link_libraries(runQuadrotorDynamics drakeRBM)
+  target_link_libraries(runQuadrotorDynamics drakeRBSystem drakeLCMSystem)
   add_dependencies(runQuadrotorDynamics drake_lcmtypes lcmtype_agg_hpp)
   pods_use_pkg_config_packages(runQuadrotorDynamics lcm)
-  add_test(NAME runQuadrotorDynamics COMMAND runQuadrotorDynamics WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/examples/Quadrotor)
```

Comment:
The previous implementation was running a 5 second simulation and thus was amenable to being a test.  This one is designed to run forever.